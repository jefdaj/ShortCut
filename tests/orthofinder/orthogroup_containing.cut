# TODO get tiny genomes to try this with like M. genitalium

paths = load_faa_each ["data/6803genes.faa", "data/7942genes.faa"]
ofres = orthofinder paths
result = orthogroup_containing ofres "Synpcc7942_0001" # TODO is this deterministic?
