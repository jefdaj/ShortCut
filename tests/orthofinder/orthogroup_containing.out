["BAA16871.1", "Synpcc7942_0001"]
